### PR TITLE
tests: do not deploy all daemons for shrink osds scenarios

### PR DESCRIPTION
**Pull request by guits**
Let's create a dedicated environment for these scenarios, there is no
need to deploy everything.
By the way, doing so will save some times.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test luminous-shrink_osd

**Comment by guits** [comment]
jenkins test luminous-shrink_osd_container

File: tests/functional/centos/7/shrink_osd/vagrant_variables.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+mds_vms: 0
+rgw_vms: 0
+nfs_vms: 0
+rbd_mirror_vms: 0
+client_vms: 0
+iscsi_gw_vms: 0
+mgr_vms: 0
+
+# Deploy RESTAPI on each of the Monitors
+restapi: true
```
Comment (leseb) [inline tests/functional/centos/7/shrink_osd/vagrant_variables.yml]: false

**Comment by leseb** [comment]
Scenarios got triggered successfully, we don't need to run the playbook, just make sure the CI triggers the scenario.

**Pull request merged**